usaspending_api/etl/transaction_loaders/field_mappings_fpds.py

```diff
@@ -506,23 +507,24 @@
 # usaspending column name -> derivation function
 recipient_location_functions = {
     "is_fpds": lambda broker: True,
+    "data_source": data_source,
     "place_of_performance_flag": lambda broker: False,
     "recipient_flag": lambda broker: True,
     "create_date": current_datetime,  # Data loader won't add this value if it's an update
     "update_date": current_datetime,
     "zip5": legal_entity_zip5,
-    "state_description": legal_entity_state_description,
```

Comment:
Both `state_name` and `state_description` are in `references_location`. I don't see `state_description` being accounted for in this dict or the nonboolean dict.

If this change stands, perhaps `legal_entity_state_description()` should be renamed.